BackofficeBundle/Controller/AdminController.php

```diff
@@ -21,8 +22,12 @@ class AdminController extends Controller
      *
      * @return Response
      */
-    public function adminAction($siteId = null)
+    public function adminAction(Request $request, $siteId = null)
     {
+        if ($request->getMethod() == 'POST') {
+            $this->get('session')->getFlashBag()->add('danger', $this->get('translator')->trans('open_orchestra_backoffice.form.javascript.error'));
```

Comment:
@alavieille , we are in context of js break and form submission. So entire page must be reloaded. I see two ways to do this, use home with a particular message created by flashbag, or a dedicated full page tpl. I choose the first one.